From 4b5c0c058ff3a7e007abfa72ce3e1d9eb140c09f Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Tue, 6 Jun 2017 13:57:13 +0800
Subject: [PATCH 1877/5242] MLK-15031-03 tty: serial: fsl_lpuart: fix the typo
 in clock get failed path

commit  0853099a8c9f49d39c267ccbcd4ae75a026c72ef from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix the typo in ipg clock get failed path.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/tty/serial/fsl_lpuart.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/fsl_lpuart.c b/drivers/tty/serial/fsl_lpuart.c
index 9d748cf..43611db 100644
--- a/drivers/tty/serial/fsl_lpuart.c
+++ b/drivers/tty/serial/fsl_lpuart.c
@@ -2354,7 +2354,7 @@ static int lpuart_probe(struct platform_device *pdev)
 
 	sport->ipg_clk = devm_clk_get(&pdev->dev, "ipg");
 	if (IS_ERR(sport->ipg_clk)) {
-		ret = PTR_ERR(sport->per_clk);
+		ret = PTR_ERR(sport->ipg_clk);
 		dev_err(&pdev->dev, "failed to get ipg clk: %d\n", ret);
 		return ret;
 	}
-- 
1.7.9.5

